cli/tsc/99_main_compiler.js

```diff
@@ -1028,7 +1028,7 @@ delete Object.prototype.__proto__;
   // and language server use it with inefficient serialization. Id is not used
   // anyway...
   function respond(id, data = null) {
-    ops.op_respond({ id, data });
+    ops.op_respond(JSON.stringify({ id, data }));
```

Comment:
Do we actually need to send the `id` here?

Reply:
No, not at the moment, we could take it out. I can see us using it if we want to do concurrent JS. requests in the future though.

Reply:
But yeah might as well remove it, and then we can just remove the whole `Response` struct

Reply:
Done ^